Makefile.js

```diff
@@ -663,11 +663,14 @@ target.gensite = function(prereleaseVersion) {
                 }
                 text += "* [Documentation source](" + docsUrl + baseName + ")\n";
             }
+
+            // 9. Update content of the file with changes
+            text.to(filename.replace("README.md", "index.md"));
```

Comment:
Ah, ok, in that case, it should work as expected, since I deleted README from rules directory